src/modules/vtol_att_control/tiltrotor.cpp

```diff
@@ -321,7 +320,7 @@ void Tiltrotor::update_transition_state()
 		// tilt rotors back once motors are idle
 		if (_time_since_trans_start > BACKTRANS_THROTTLE_DOWNRAMP_DUR_S) {
 
-			float progress = (_time_since_trans_start - BACKTRANS_THROTTLE_DOWNRAMP_DUR_S) / BACKTRANS_MOTORS_UPTILT_DUR_S;
+			float progress = (_time_since_trans_start - BACKTRANS_THROTTLE_DOWNRAMP_DUR_S) / _param_vt_bt_tilt_dur.get();
```

Comment:
@sfuhrer Should we add some protection here, in case someone sets this to 0?

Reply:
good idea, added a max(param, 0.1)